astropy/stats/tests/test_sigma_clipping.py

```diff
@@ -430,7 +441,7 @@ def test_sigma_clip_axis_shapes(axis, bounds_shape):
     assert bound2.shape == bounds_shape
 
 
-@pytest.mark.parametrize('dtype', ['>f2', '<f2', '>f4', '<f4', '>f8', '<f8', '<i4', '>i8'])
+@pytest.mark.parametrize('dtype', ['>f4', '<f4', '>f8', '<f8', '<i4', '>i8'])
```

Comment:
Maybe try whether just setting the data element to 100 instead of 1000 would mean things would also work for `f2`?

Reply:
Yes, that works.  I added back the float16 tests.